File: 프로그래머스/1/42748. K번째수/K번째수.js
function solution(array, commands) {
    const result = []
    for (const [i,j,k] of commands){
        let item_list = i === j ? array.slice(i-1,i) : array.slice(i - 1,j)
        item_list = item_list.sort((a,b) => a- b)
        console.log(item_list)
        result.push(item_list[k - 1])
    }
    return result
}

File: 프로그래머스/1/42748. K번째수/K번째수.cpp
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

vector<int> solution(vector<int> array, vector<vector<int>> commands) {
    vector<int> answer;
    for (int l = 0; l < commands.size(); l++){
        int i = commands[l][0];
        int j = commands[l][1];
        int k = commands[l][2];
        vector<int> arr;
        for (int m = i - 1; m < j; m++){
            arr.push_back(array[m]);
        }
        sort(arr.begin(),arr.end());
        answer.push_back(arr[k - 1]);
    }
    return answer;
}